blog-posts/emails/article.md

```diff
@@ -0,0 +1,331 @@
+---
```

Comment:
Just a general comment, even if it sounds great in french, using plural 1st pronom is not common in english. Consider replacing "our/us" by "mine/I" or addressing the reader directly with "your/you"